Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -77,4 +117,75 @@ public function getPriority()
         // should be run after the UnusedUseFixer
         return -20;
     }
+
+    private function removeBetweenUse($index)
+    {
+        // remove empty lines between two `use`s
+        $tokenCount = count($this->tokens);
+        for ($i = $index; $i < $tokenCount; ++$i) {
+            if (';' === $this->tokens[$i]->getContent()) {
+                $next = $this->tokens->getNextMeaningfulToken($i);
+                if ($this->tokens[$next]->isGivenKind(T_USE)) {
+                    for ($i = $next; $i > $index; --$i) {
+                        if ($this->tokens[$i]->isWhitespace() && substr_count($this->tokens[$i]->getContent(), "\n") > 1) {
+                            $this->tokens[$i]->setContent("\n");
+                        }
+                    }
+                }
+
+                break;
+            }
+        }
+    }
+
+    private function removeMultipleBlankLines($index)
+    {
+        $token = $this->tokens[$index];
+        $content = '';
+        $count = 0;
+        $parts = explode("\n", $token->getContent());
+
+        for ($i = 0, $last = count($parts) - 1; $i <= $last; ++$i) {
+            if ('' === $parts[$i] || "\r" === $parts[$i]) {
```

Comment:
Introduced on 2.x line: `Symfony\CS\Tokenizer\Transformer\WhitespacyCommentTransformer`.
I think this is what you asked for.